addons/l10n_ar/models/account_journal.py

```diff
@@ -107,12 +107,11 @@ def _get_journal_codes(self):
                     'l10n_latam_use_documents')
     def _check_afip_configurations(self):
         """ Do not let to update journal if already have confirmed invoices """
```

Comment:
maybe improve this comment meanwhile
"Do not let the user update the journal if it already contains confirmed invoices"

Reply:
Ok, sounds better, i'll change it